src/main/java/com/pd/algorithms/CountSorting.java

```diff
@@ -0,0 +1,69 @@
+package sorting;
+
+import java.util.Scanner;
+
+public class CountSorting {
+    static void countSort(int n, int... arr){
+        //1.Find out the maximum element (let it be max) from the given array.
+        int max=0;
+        for(int i=0;i< arr.length;i++){
+            if(max<arr[i]){
+                max=arr[i];
+            }
+
+        }
+        //2.Initialize an array of length max+1 with all elements 0.
+        // This array is used for storing the count of the elements in the array.
+        int[] count= new int[max+1];
+
+        //3.Store the count of each element at their respective index in count array.
+        for(int i=0;i<arr.length;i++) {
+            count[arr[i]]++;
+        }
+        //4.Store cumulative sum of the elements of the count array.
+        // It helps in placing the elements into the correct index of the sorted array.
+        for(int i=1;i<count.length;i++){
+            count[i]=count[i]+count[i-1];
+        }
+        //5.Find the index of each element of the original array in count array, and
+        // place the elements in output array
+        int[] arr_sorted= new int[n];
+        for(int i=n-1; i>=0; i--) {
+            arr_sorted[count[arr[i]] - 1] = arr[i];
+            count[arr[i]]--;
+        }
+        //6.Copy the sorted elements into original array
+        System.out.println("SORTED ARRAY :");
+        for(int i=0; i<n; i++){
+            arr[i]=arr_sorted[i];
+            System.out.print(" "+arr[i]);
+        }
+
+    }
+    public static void main(String[] args) {
```

Comment:
We don't really need a `main` method for the class. Also, please write tests for your method.